pkg/kclient/secrets.go

```diff
@@ -15,8 +13,7 @@ import (
 // portNumber is the target port of the ingress
 func (c *Client) CreateTLSSecret(tlsCertificate []byte, tlsPrivKey []byte, componentName string, applicationName string, portNumber int) (*corev1.Secret, error) {
 	labels := componentlabels.GetLabels(componentName, applicationName, true)
-	portAsString := fmt.Sprintf("%v", portNumber)
```

Comment:
why is port number removed? If it not required anymore please remove the portNumber from the function signature and also the function comment

Reply:
updated